asciidoc/courses/llm-vectors-unstructured/modules/3-unstructured-data/lessons/7-knowledge-graph/lesson.adoc

```diff
@@ -0,0 +1,26 @@
+= Turning data into knowledge 
+:order: 7
+:type: lesson
+
+During this course, you have learned how you can Neo4j to turn knowledge into data, the first step in creating a knowledge graph.
+Knowledge graphs are a powerful way to represent and query complex interconnected data.
+
+Knowledge graphs are essential to many AI and machine learning applications. 
+You can use a knowledge graph to give context and ground an LLM, giving it access to structured data beyond its initial training data.
+
+You can find more information about knowledge graphs including white papers, and free resources at link:https://neo4j.com/use-cases/knowledge-graph[neo4j.com/use-cases/knowledge-graph^].
+
+== Next Steps
+
+You can learn more about how to use Neo4j and LLMs in other GraphAcademy courses:
+
+* link:https://graphacademy.neo4j.com/courses/llm-fundamentals/[Neo4j & LLM Fundamentals^]
+* link:https://graphacademy.neo4j.com/courses/llm-chatbot-python/[Build a Neo4j-backed Chatbot using Python^]
+* link:https://graphacademy.neo4j.com/courses/llm-chatbot-typescript/[Build a Neo4j-backed Chatbot with TypeScript^]
+
+read::Continue[]
```

Comment:
A lot of these say "Continue" rather than something more descriptive like "Mark as read"